client_tools.go

```diff
@@ -67,6 +67,80 @@ func (o Op) EncodeMsgpack(enc *msgpack.Encoder) error {
 	return enc.Encode(o.Arg)
 }
 
+// We don't want to make it public.
```

Comment:
I think comment is not needed here. All constants with lower first letter are not public.

Reply:
Okay, fixed.